Aquib/1_Valid_Sudoku.c.txt

```diff
@@ -0,0 +1,83 @@
+bool checkRow(char **board, int n)
```

Comment:
Let's add high level comments to get in the habit of adding comments and making the solution self explanatory.

Reply:
Yes. I shall add comments to the code and recommit it. I tried to make the code more readable by the variable names itself.

Reply:
sg